src/pat/autotoc/autotoc.js

```diff
@@ -1,5 +1,6 @@
 import $ from "jquery";
 import Base from "@patternslib/patternslib/src/core/base";
+import utils from "@patternslib/patternslib/src/core/utils";
```

Comment:
Oh, imported but unused ... I want to use utils.debounce to debounce the tabs-update logic and not run for every input-check.

Still TO DO.

Reply:
done